edit-post/components/meta-boxes/unsaved-warning.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { some } from 'lodash';
+import jQuery from 'jquery';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { getMetaBoxes } from '../../store/selectors';
+import { getMetaBoxContainer } from '../../utils/meta-boxes';
+
+class MetaBoxesUnsavedWarning extends Component {
+	/**
+	 * @inheritdoc
```

Comment:
Do we care for these JSDoc? I'm guessing from copy paste of underlying `UnsavedChangesWarning`.

Speaking of, it would be nice it this wasn't so non-DRY 😬

Reply:
Yep, I was wondering if I should create another component or provide an extensibility APi or a prop or something for the other one, I thought this one is simpler to understand, but I can update to introduce a callaback to enhance the `isDirty` check.